src/communication/communication/csv_parse.py

```diff
@@ -0,0 +1,127 @@
+"""
+CSV Parsing Node that will parse the CSV files and send information to topics.
+Subscibers that need the CSV information and the React GUI will subscribe to these topics.
+"""
+import json
+import rclpy
+from rclpy.node import Node
+import pandas as pd
```

Comment:
pandas isnt native to python, i will read about how to deploy a venv on the jetson